packages/react-native-codegen/src/parsers/flow/components/props.js

```diff
@@ -46,13 +51,29 @@ function buildPropSchema(
   };
 }
 
+/**
+ * Returns the properties of a type definition, including those from
+ * `extends` clauses.
+ * @param typeDefinition The type definition to get properties from.
+ * @param types The type declarations.
+ */
 function getProps(
   typeDefinition: $ReadOnlyArray<PropAST>,
   types: TypeDeclarationMap,
-): $ReadOnlyArray<NamedShape<PropTypeAnnotation>> {
-  return flattenProperties(typeDefinition, types)
+): {
+  props: $ReadOnlyArray<NamedShape<PropTypeAnnotation>>,
+  extendsProps: $ReadOnlyArray<ExtendsPropsShape>,
+} {
+  const props = flattenProperties(typeDefinition, types)
     .map(property => buildPropSchema(property, types))
     .filter(Boolean);
+
+  const extendsProps = typeDefinition
+    .filter(prop => prop.type === 'ObjectTypeSpreadProperty')
+    .map(prop => extendsForProp(prop, types))
+    .filter(Boolean);
```

Comment:
This code is copied, from `getExtendsProps`, right? why do not move the function and use it, rather than copying the code?

Reply:
Sure. I though we're \*actually* *merging* the functions :)
Also, the having fixtures makes it little difficult for checking where & what the tests actually are... ;)